atd-vze/src/views/Crashes/Recommendations/RecommendationTextInput.js

```diff
@@ -42,7 +44,7 @@ const RecommendationTextInput = ({
   };
 
   // Recommendation record does not exist yet, adding a value will create record
-  const isAddingRecommendation =
+  const isCreatingRecord =
```

Comment:
make it more apparent that we are talking about this is going to create a recommendation record, not talking about just adding recommendation text